codec.go

```diff
@@ -114,15 +114,15 @@ func (u *UUID) UnmarshalText(text []byte) error {
 	case 41, 45:
 		return u.decodeURN(text)
 	default:
-		return fmt.Errorf("uuid: incorrect UUID length: %s", text)
+		return fmt.Errorf("uuid: incorrect UUID length %d in string '%s'", len(text), text)
```

Comment:
What if we used `%q` instead of `'%s'`?

Reply:
I actually wasn't aware of the `%q` verb. That's neat! I switched the code to use `%q` instead of `'%s'`